docs/source/tutorials_source/platform/tutorial_cropped_objects_metadata.py

```diff
@@ -0,0 +1,166 @@
+    bounding_boxes_list_list = []
+    for prediction in predictions:
+        bounding_boxes_list = []
+        for box in prediction["boxes"]:
+            x0 = box[0] / x_size
+            y0 = box[1] / y_size
+            x1 = box[2] / x_size
+            y1 = box[3] / y_size
+            bounding_boxes_list.append(BoundingBox(x0, y0, x1, y1))
+        bounding_boxes_list_list.append(bounding_boxes_list)
+
+    cropped_images_list_list = crop_dataset_by_bounding_boxes_and_save(dataset, OUTPUT_DIR,
+        bounding_boxes_list_list, class_indices_list_list)
+
+    '''  5. Save the objectness scores as metadata '''
+    objectness_scores_list_list = [list(prediction["scores"]) for prediction in predictions]
+    metadata_list = []
+    for cropped_images_list, objectness_scores_list in zip(cropped_images_list_list, objectness_scores_list_list):
+        for cropped_images_filename, objectness_score in zip(cropped_images_list, objectness_scores_list):
+            metadata = {"objectness_score": float(objectness_score)}
+            metadata_list.append((cropped_images_filename, metadata))
+    save_custom_metadata(METADATA_OUTPUT_FILE, metadata_list)
+
+    ''' 6. Tell the lightly CLI command '''
+    cli_command = f"lightly-magic input_dir={OUTPUT_DIR} new_dataset_name=SKU_110k_val_cropped trainer.max_epochs=0 "
+    cli_command += f"custom_metadata={METADATA_OUTPUT_FILE} token=MY_TOKEN"
+    print(f"Upload the images and custom metadata with the following CLI command:")
+    print(f"{bcolors.OKBLUE}{cli_command}{bcolors.ENDC}")
+
+6. Adapt this command to include your token and run it in a terminal.
+You can also change some arguments, e.g. to train a better embedding model instead of relying on
+a pretrained one. For more information, head to :ref:`lightly-command-line-tool`.
+
+7. Once the cropped images, embeddings and metadata are uploaded, we can use the Lightly Webapp
+to configure the Objectness Score as metadata. This is done in the Configuration view.
+
+.. figure:: ../../tutorials_source/platform/images/tutorial_cropped_objects_metadata/sku110k_config_metadata.png
+    :align: center
+    :alt: Configuration of Objectness Score as metadata
+
+8. Now we can switch to the explore view and select to sort by the Objectness Score in ascending order.
+We directly see that many images show images despite having a low objectness score,
+thus they are false negatives / missing examples.
+
+.. figure:: ../../tutorials_source/platform/images/tutorial_cropped_objects_metadata/sku110k_explore_sort_objectness.png
+    :align: center
+    :alt: Explore view of images sorted by ascending objectness score
+
+When clicking on one them, we wee that they it has a low objectness score of only 0.05, despite showing an object,
+thus it is false negative. Similar images which are also false negatives are shown as well.
+Thus all of them can be added directly to the list of missing examples,
```

Comment:
How can we add them to the list of missing examples? I think here we could also show a gif on how we add similar images

Reply:
This feature is missing in the webapp.

Reply:
I added the screenshots + description.